requirements.txt

```diff
@@ -67,160 +44,74 @@ flake8-return==1.2.0 ; python_version >= "3.8" and python_version < "3.12"
-prometheus-client==0.15.0 ; python_version >= "3.8" and python_version < "3.12"
-prompt-toolkit==3.0.32 ; python_version >= "3.8" and python_version < "3.12"
-psutil==5.9.4 ; python_version >= "3.8" and python_version < "3.12"
 ptable==0.9.2 ; python_version >= "3.8" and python_version < "3.12"
-ptyprocess==0.7.0 ; python_version >= "3.8" and python_version < "3.12" and sys_platform != "win32" or python_version >= "3.8" and python_version < "3.12" and os_name != "nt"
-pure-eval==0.2.2 ; python_version >= "3.8" and python_version < "3.12"
-py==1.11.0 ; python_version >= "3.8" and python_version < "3.12"
-pybufrkit==0.2.19 ; python_version >= "3.8" and python_version < "3.12"
 pycodestyle==2.8.0 ; python_version >= "3.8" and python_version < "3.12"
 pycparser==2.21 ; python_version >= "3.8" and python_version < "3.12"
 pydocstyle==6.1.1 ; python_version >= "3.8" and python_version < "3.12"
 pyflakes==2.4.0 ; python_version >= "3.8" and python_version < "3.12"
 pygments==2.13.0 ; python_version >= "3.8" and python_version < "3.12"
-pyopenssl==22.1.0 ; python_version >= "3.8" and python_version < "3.12"
 pyparsing==3.0.9 ; python_version >= "3.8" and python_version < "3.12"
 pypdf2==1.28.6 ; python_version >= "3.8" and python_version < "3.12"
-pyrsistent==0.19.2 ; python_version >= "3.8" and python_version < "3.12"
-pysocks==1.7.1 ; python_version >= "3.8" and python_version < "3.12"
-pytest-cov==3.0.0 ; python_version >= "3.8" and python_version < "3.12"
-pytest-dictsdiff==0.5.8 ; python_version >= "3.8" and python_version < "3.12"
-pytest-forked==1.4.0 ; python_version >= "3.8" and python_version < "3.12"
-pytest-notebook==0.7.0 ; python_version >= "3.8" and python_version < "3.12"
-pytest-xdist==2.5.0 ; python_version >= "3.8" and python_version < "3.12"
-pytest==7.2.0 ; python_version >= "3.8" and python_version < "3.12"
 python-dateutil==2.8.2 ; python_version >= "3.8" and python_version < "3.12"
 python-dotenv==0.21.0 ; python_version >= "3.8" and python_version < "3.12"
 pytz-deprecation-shim==0.1.0.post0 ; python_version >= "3.8" and python_version < "3.12"
 pytz==2022.6 ; python_version >= "3.8" and python_version < "3.12"
-pywin32==305 ; sys_platform == "win32" and platform_python_implementation != "PyPy" and python_version >= "3.8" and python_version < "3.12"
-pywinpty==2.0.9 ; python_version >= "3.8" and python_version < "3.12" and os_name == "nt"
 pyyaml==6.0 ; python_version >= "3.8" and python_version < "3.12"
-pyzmq==24.0.1 ; python_version >= "3.8" and python_version < "3.12"
 rapidfuzz==2.13.2 ; python_version >= "3.8" and python_version < "3.12"
 regex==2022.3.2 ; python_version >= "3.8" and python_version < "3.12"
 requests==2.28.1 ; python_version >= "3.8" and python_version < "3.12"
-requests[security]==2.28.1 ; python_version >= "3.8" and python_version < "3.12"
-ruff==0.0.46 ; python_version >= "3.8" and python_version < "3.12"
+ruff==0.0.120 ; python_version >= "3.8" and python_version < "3.12"
```

Comment:
Why is `ruff` still part of the `requirements.txt`?

Reply:
I see. The export to `requirements.txt` includes development dependencies. Is it intended like this? If the Anaconda packages will derive its runtime dependencies from this list, maybe we should change it? See also https://github.com/conda-forge/wetterdienst-feedstock/issues/57.

In order to provide a corresponding file _including_ the development dependencies, we can use `requirements-dev.txt`.

```
export_requirements = "poetry export --without-hashes --with=dev --output requirements.txt"
```

Reply:
See #790.